tests/src/Interop/COM/ServerContracts/Server.Contracts.h

```diff
@@ -163,17 +163,6 @@ INumericTesting : IUnknown
         /*[in]*/ int i11,
         /*[in]*/ int i12,
         /*[out]*/ int * result ) = 0;
-        virtual COM_DECLSPEC_NOTHROW SizeF STDMETHODCALLTYPE MakeSize(
```

Comment:
You should also be able to get rid of the `SizeF` and `Size` struct declarations.

Reply:
Same in the managed contract definitions.